tests/manager/api/admin/controller/test_custom_lists.py

```diff
@@ -495,7 +498,10 @@ def test_custom_list_get_with_pagination(
             work = admin_librarian_fixture.ctrl.db.work(with_license_pool=True)
             list.add_entry(work)
 
-        with admin_librarian_fixture.request_context_with_library_and_admin("/"):
+        with (
+            admin_librarian_fixture.request_context_with_library_and_admin("/"),
+            services_fixture.wired(),
+        ):
```

Comment:
This change is needed because without a wired service container we get an exception (and have been for awhile) that was being swallowed by the except clause I removed.